### PR TITLE
Test case page 2. Choosing a male head 1

### LINKED ISSUE #25: Test case page 2. Choosing a male head 1
1 Choose a male figure
2 Select head 1
3 Compare the reference picture with the total

### PR DESCRIPTION
**Pull request by JuliaMetel**

### CODE REVIEW

File: page_two.py [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+        return FoundationElement((By.ID, 'eye'), self._driver)
+
+    def element_head_1(self):
+        return FoundationElement((By.ID, 'head_1'), self._driver)
+
+    def wait_canvas_stable(self):
+        canvas_rect = self.element_canvas().get_rect()
+        for x in range(5):
+            image_1 = self.create_screenshot_by_coordinates(canvas_rect)
+            if is_images_equal(image_1, Image.open("Test_screenshots/test_decals_head_1_male.png")):
```
Comment (IamVladislav) [inline page_two.py]: Bad hardcode. Take this as param from test

File: page_two.py [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+                return True
```
Comment (IamVladislav) [inline page_two.py:24]: Waiter shouldn't return something. 
Please, rename it to is_canvas_equal

File: page_two.py (hunk 1 above)
Comment (JuliaMetel) [reply page_two.py]: agree, fixed

File: page_two.py (hunk 2 above)
Comment (JuliaMetel) [reply page_two.py:24]: Fixed

**Review by IamVladislav — APPROVED** [review]

**Pull request merged**